server/odc-common/src/main/java/com/oceanbase/odc/common/util/SystemUtils.java

```diff
@@ -115,6 +117,16 @@ public static Map<String, Object> getSystemMemoryInfo() {
         return metric2Messages;
     }
 
+    /**
+     * get system free physical memory, unit size is MB
+     *
+     * @return free memory
+     */
+    public static long getSystemFreeMemory() {
```

Comment:
you may consider BinarySize as the return value or you can change your method name: getSystemFreeMemorySizeMilliBytes